config/locales/server.en.yml

```diff
@@ -2080,6 +2082,9 @@ en:
 
         If the above link is not clickable, try copying and pasting it into the address bar of your web browser.
 
+    push:
+      body: You have %{count} new notifications.
```

Comment:
hmm I could add that in but currently, we won't call the translation when we only have one notification.